### PR TITLE
fix: set version of notebooks automatically to latest

### PR DESCRIPTION
**Pull request by jcabrero**

### CODE REVIEW

File: examples/complex_model/complex_model.ipynb [hunk 1]
```diff
@@ -60,7 +60,7 @@
       },
       "outputs": [],
       "source": [
-        "%pip install nada-ai~=0.3.0 --quiet"
+        "%pip install nada-ai --quiet"
```
Comment (mathias-nillion) [inline examples/complex_model/complex_model.ipynb:63]: @jcabrero should we instead also specify a version for nada? Here we implicitly assume that nada-ai:latest will always work with latest nada-dsl

**Review by mathias-nillion — APPROVED** [review]
LGTM - left a small thought. LMK what you think

File: examples/complex_model/complex_model.ipynb (hunk 1 above)
Comment (jcabrero) [reply examples/complex_model/complex_model.ipynb:63]: That's a hard question. There are pros/cons for each of those and it's hard to tell. I would argue that always being par with nada-dsl is what requires the least work as the API should slowly settle down. However, if we fix versions, every two weeks (or once there's a new release) we need to dedicate a whole morning at least to update all the versions manually and verify that everything works. Not sure what's best here.

Comment (jcabrero) [reply examples/complex_model/complex_model.ipynb:63]: I am going to merge this for now, but it's a good discussion to have and see what's best.

**Pull request merged**